compiler/scenario-service/server/src/main/scala/com/digitalasset/daml/lf/scenario/Context.scala

```diff
@@ -246,16 +249,15 @@ class Context(
         e.cause match {
           case e: Error => handleFailure(e)
           case e: speedy.SError.SError => handleFailure(Error.RunnerException(e))
-          case e => {
-            // We can't send _everything_ over without changing internal, nicer to put a print here.
-            e.printStackTrace
+          case e =>
+            // We can't send _everything_ over without changing internal, we log and wrap the error in t.
+            logger.warn("Script.FailedCmd unexpected cause: " + e.getMessage)
```

Comment:
Is there nowhere else we can put the trace? maybe logger.debug?

Reply:
3e99b5a9402ce550d4e191fb9c607054e90d437f